src/views/Home.vue

```diff
@@ -4,37 +4,71 @@
     p(class="subtitle") BOOKS
     form(class="container-form" @submit.prevent="onSubmit")
       label(class="label-input") First name
-      input(type="text", class="input-primary" v-model="firstName")
+      input(type="text", class="input-primary", v-model="$v.form.firstName.$model")
+      div(v-if="$v.form.firstName.$error")
+        span(class="alert" class="alert" v-if="!$v.form.firstName.required") This field is required
 
       label(class="label-input") Last name
-      input(type="text", class="input-primary" v-model="lastName")
+      input(type="text", class="input-primary" v-model="$v.form.lastName.$model")
+      div(v-if="$v.form.lastName.$error")
+        span(class="alert" class="alert" v-if="!$v.form.lastName.required") This field is required
 
       label(class="label-input") Email
```

Comment:
At wolox we use it on this way
```suggestion
      label(class="label-input")
      | Email
```